README.md

```diff
@@ -57,6 +57,32 @@ Get in touch with us on [Slack](https://aif360.slack.com) (invitation [here](htt
 
 Installation is easiest on a Unix system running Python 3. See the additional instructions for [Windows](#windows) and [Python 2](#python-2) as appropriate.
 
+### (Optional) Create a Virtualenv environment
```

Comment:
Lets mention that these instructions are for Unix/MacOS (since ~ wont work in Windows). source will also not work.

Reply:
okay, I added additional instructions for Windows